src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -1477,14 +1477,20 @@ protected override void HandleStackTraceRequestAsync(IRequestResponder<StackTrac
                             }
                         }
 
+                        IDebugCodeContext2 memoryAddress = null;
+                        frame.GetCodeContext(out memoryAddress);
+                        var contextInfo = new CONTEXT_INFO[1];
+                        memoryAddress.GetInfo(enum_CONTEXT_INFO_FIELDS.CIF_ALLFIELDS, contextInfo); // test -- need to delete; need to check if call is successful
```

Comment:
No need to grab all fields, you can just ask for the Address. You will want to validate that the contextInfo also set the flag that it has a value. E.g.
`contextInfo[0].dwFields.HasFlag(enum_CONTEXT_INFO_FIELDS.CIF_ADDRESSABSOLUTE)` before using `contextInfo[0].bstrAddressAbsolute`